pkgs/by-name/wi/wivrn/package.nix

```diff
@@ -0,0 +1,123 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, vulkan-headers
+, vulkan-loader
+, x264
+, xorg
+, eigen
+, boost
+, glm
+, spdlog
+, monado
+, openxr-loader
+, nlohmann_json
+, harfbuzz
+, freetype
+, glslang
+, python3
+, udev
+, ffmpeg
+, libva
+, libdrm
+, avahi
+, libpulseaudio
+, pkgs
+}:
+let
+  # We need tiny_gltf.h and draco doesn't provide it
+  tinygltf = pkgs.callPackage "${pkgs.path}/pkgs/development/libraries/draco/tinygltf.nix" {};
```

Comment:
There's `draco = callpackage ... { tinygltf = callPackage ... }` in `all-packages.nix`. Now that there's more than 1 package using `tinygltf` it's only reasonable to introduce a top-level attribute for it?